### DISCUSSION TITLE
Add percentage, time and counts in State Timeline

### DISCUSSION THREAD

**Discussion by busi91** [grafana/grafana#53955] [Dashboards, Panels & Visualizations]
Hi,
In order to replace the Discrete Panel (poorly supported in latest Grafana version) with the new core panel State Timeline, I think will be usefull to add legend basic calculations, such as percentage, time and counts (like disctete panel options).

![image](https://user-images.githubusercontent.com/42997776/185632751-3b6c67c4-e402-4336-aa98-d32b0b1e9290.png)

Here is the difference between the two panels: 

![image](https://user-images.githubusercontent.com/42997776/185632907-1024401d-956c-4dfa-ab77-94b54b8744e9.png)

**Comment by rvdwijngaard** [comment]
This is definitely something which would make this visualization much better.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
